android/.idea/markdown-exported-files.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is the .idea folder supposed to be committed?

Reply:
I just used the default .gitignore that android studio created. It only emitted some of the .idea directory and kept some so android studio could easily allow someone to "open an existing project." Would you prefer me to remove it completely?

Reply:
Cool beans